components/generic/ShareButton.vue

```diff
@@ -0,0 +1,31 @@
+<template>
```

Comment:
Let's move the three sharing-related components to components/sharing

Reply:
ok shareButton, socialShare and socialShareModal have been to the new sharing folder